PYME/experimental/isosurface.py

```diff
@@ -25,4 +26,125 @@ def isosurface(data, isolevel, voxel_size=None, origin=None, remesh=False):
     if remesh:
         T.remesh()
     
-    return T
\ No newline at end of file
+    return T
+
+def dot(v, w):
```

Comment:
why the redefinition of a bunch of functions which are available in numpy?

Reply:
We use this redefinition of `dot` all over the triangle_mesh stuff because it's faster than `np.dot` for vectors of length `(3,)`. I just formalized it here. We don't even use it, though, so I'll delete it.